lib/grape/validations/validate_service.rb

```diff
@@ -0,0 +1,51 @@
+module Grape
+  module Validations
+    class ValidateService
+      def initialize(validators, params)
+        @validators = validators
+        @params = params
+      end
+
+      # api
+      def validate!
```

Comment:
This method implies that an exception is going to be raised (it has a bang), but instead it returns a boolean. So it should either be renamed to `validate` or it should `fail ...`.